addons/mass_mailing/tests/test_mailing_internals.py

```diff
@@ -62,6 +66,89 @@ def test_mailing_body_responsive(self):
         self.assertIn('@media', body_html)
         self.assertIn('I am Responsive body', body_html)
 
+    @users('user_marketing')
+    def test_mailing_body_inline_image(self):
+        attachments = []
+        original_image_to_url = self.env['mailing.mailing']._image_to_url
+        def patched_image_to_url(self, b64image: bytes):
```

Comment:
If ``_image_to_url`` returns the new attachments, it would be way easier to mock it ^^

Reply:
Ok but then we have to do the creation of the URL manually every time we call it...?